reference/OpenAPI.json

```diff
@@ -8409,6 +8412,81 @@
           "code"
         ]
       },
+      "AccessSettingsAssignmentsList": {
+        "title": "Access Settings Assignments List",
+        "type": "object",
+        "description": "Lists all assignments of the campaign to areas and stores if the Areas and Stores feature is enabled (Enterprise feature).",
+        "properties": {
+          "object": {
+            "type": "string",
+            "default": "list",
+            "description": "The type of the object represented by JSON. Default is `list`. This object stores information about campaign assignments to areas and stores",
+            "enum": [
+              "list"
+            ]
+          },
+          "data_ref": {
+            "type": "string",
+            "default": "data",
+            "description": "Identifies the name of the attribute that contains the array of campaign assignments.",
+            "enum": [
+              "data"
+            ]
+          },
+          "data": {
+            "type": "array",
+            "description": "Contains an array of campaign assignments.",
+            "items": {
+              "type": "object",
```

Comment:
To powinien być osobny obiekt

Reply:
Osobny schemat którego można użyć gdzieś indziej. Nazwany być może `areaStoreCampaignAssignment` ale to warto byłoby jeszcze sprawdzić.

Reply:
Dodałem osobny obiekt.

W tej PR-ce jeszcze dołożę link w opisie do artykułu o Areas i Stores, gdy powstanie i będzie opublikowany.